Header header
DroneWaypointStamped[] trajectory
int32 num_node
int32 num_segment